pupil-spa/src/app/services/route/route.service.ts

```diff
@@ -9,6 +10,10 @@ export class RouteService {
   }
 
   public setup(): void {
+    if (APP_CONFIG.websiteOffline) {
+      this.router.navigate(['/service-unavailable'])
```

Comment:
@danwilliams wouldn't it be better if the router navigation occurred within `app.component.js`?
Potentially after the route service setup call?
I think this shouldn't belong in the setup since the behavior is related to the availability of the app so the master component should be handling it.

Reply:
I agree - changed 👍